libmysqlclient.18.dylib

```diff
@@ -0,0 +1 @@
+/usr/local/mysql/lib/libmysqlclient.18.dylib
```

Comment:
oh, is this required?

Reply:
@jywarren macOS Mojave searches for this. It's configurations for mysql are a little different than any past system with macOS. It is not a file it's a sym link. Maybe @gauravano can double check that this doesn't effect any other users?

Reply:
@jywarren sorry this is an issue for #355. Should I just remove all changes related to #355 out from under this? It was just a little easier for me to work with the same configs across branches but if it causes confusion it's not worth it.

Reply:
Tried this PR locally, no issue due to this.